src/main/java/org/folio/fqm/service/CrossTenantQueryService.java

```diff
@@ -79,12 +83,24 @@ public List<String> getTenantsToQuery(EntityType entityType) {
 
   public String getCentralTenantId() {
     try {
-      String rawJson = ecsClient.get("consortia-configuration", Map.of());
+      String rawJson = ecsClient.get(CONSORTIA_CONFIGURATION_PATH, Map.of());
       DocumentContext parsedJson = JsonPath.parse(rawJson);
-      return parsedJson.read("centralTenantId");
+      return parsedJson.read(CENTRAL_TENANT_ID);
     } catch (Exception e) {
-      log.debug("Error retrieving tenants for cross-tenant query. Tenant may not be in an ECS environment.");
+      log.debug(CROSS_TENANT_QUERY_ERROR);
       return null;
     }
   }
+
+  public boolean ecsEnabled() {
+    try {
+      String rawJson = ecsClient.get(CONSORTIA_CONFIGURATION_PATH, Map.of());
+      DocumentContext parsedJson = JsonPath.parse(rawJson);
+      parsedJson.read(CENTRAL_TENANT_ID);
+      return true;
+    } catch (Exception e) {
+      log.debug(CROSS_TENANT_QUERY_ERROR);
+      return false;
+    }
+  }
```

Comment:
This essentially duplicates a method we have in the EntityTypeFlatteningService, but it makes more sense to have here. I was going to replace the method in EntityTypeFlatteningService with a call to this one, but that leads to a circular dependency that would be more of a headache than it's worth dealing with. So I think for now it's probably ok to have it in both spots, though this is something I'd want to clean up in the future.